pkg/queue/manager.go

```diff
@@ -284,7 +284,7 @@ func (m *Manager) addOrUpdateWorkload(w *kueue.QueuedWorkload) bool {
 // RequeueWorkload requeues the workload ensuring that the queue and the
 // workload still exist in the client cache and it's not admitted. It won't
 // requeue if the workload is already in the queue (possible if the workload was updated).
-func (m *Manager) RequeueWorkload(ctx context.Context, info *workload.Info) bool {
+func (m *Manager) RequeueWorkload(ctx context.Context, info *workload.Info, skipped bool) bool {
```

Comment:
lets unify the interface, flip the flag semantics and name it `inadmissible` like  `RequeueIfNotPresent`

Reply:
Done